File: code/packages/fsharp/lattice-parser/tests/CodingAdventures.LatticeParser.Tests/LatticeParserTests.fs
namespace CodingAdventures.LatticeParser.Tests

open CodingAdventures.LatticeParser.FSharp
open Xunit

module LatticeParserTests =
    [<Fact>]
    let pingReturnsPackageName () =
        Assert.Equal("lattice-parser", LatticeParser().Ping())

File: code/packages/csharp/lattice-parser/tests/CodingAdventures.LatticeParser.Tests/LatticeParserTests.cs
namespace CodingAdventures.LatticeParser.Tests;

public sealed class LatticeParserTests
{
    [Fact]
    public void PingReturnsPackageName()
    {
        Assert.Equal("lattice-parser", new CodingAdventures.LatticeParser.LatticeParser().Ping());
    }
}